client/my-sites/people/people-list-section-header/test/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'PeopleListSectionHeader', function() {
+	let configMock,
+		PeopleListSectionHeader,
+		SectionHeader;
```

Comment:
there's some indenting that needs fixing here. if it doesn't show up in the linting then we need to fix the linting